src/libopenrave/kinbodygrab.cpp

```diff
@@ -684,6 +768,9 @@ void KinBody::GrabbedInfo::serialize(std::ostream& o) const
     for( std::set<std::string>::const_iterator it = _setIgnoreRobotLinkNames.begin(); it != _setIgnoreRobotLinkNames.end(); ++it ) {
         o << (*it) << " ";
     }
+    if( !!_prUserData ) {
+        o << OpenRAVE::orjson::DumpJson(*_prUserData) << " ";
```

Comment:
Can you use the ``DumpJson`` overload which takes ``std::ostream`` as parameter so that ``std::setprecision`` may be respected better?

Reply:
I changed to use `ostream` overload with a comment in the code.